contracts/RiskSteward/RiskStewardReceiver.sol

```diff
@@ -136,12 +141,13 @@ contract RiskStewardReceiver is
         if (debounce == 0) {
             revert InvalidDebounce();
         }
+        RiskParamConfig memory previousConfig = riskParameterConfigs[updateType];
         riskParameterConfigs[updateType] = RiskParamConfig({
             active: true,
             riskSteward: riskSteward,
             debounce: debounce
         });
-        emit RiskParameterConfigSet(updateType, riskSteward, debounce);
+        emit RiskParameterConfigSet(updateType, previousConfig.riskSteward, riskSteward, previousConfig.debounce, debounce, previousConfig.active, true);
```

Comment:
It's ok, but if you emit the event before changing the stored value, you will save some gas

Reply:
Maybe I'm wrong, because in this case the storage variable is a struct, so using a temporary variable saves gas